client/src/js/updates/components/__tests__/Release.test.js

```diff
@@ -1,27 +1,33 @@
-import { Col } from "react-bootstrap";
 import Release, { ReleaseMarkdown } from "../Release";
 
 describe("<Release />", () => {
     let props;
-    let wrapper;
-
+    let state;
     it("renders correctly", () => {
         props = {
             body: "foo bar baz",
             name: "test",
             html_url: "www.github.com/virtool/virtool"
         };
-        wrapper = shallow(<Release {...props} />);
+        const wrapper = shallow(<Release {...props} />);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['wrapper' is assigned a value but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5026542)